src/commands.ts

```diff
@@ -195,7 +196,11 @@ export const canInsert =
     const { $from, $to } = state.selection
 
     // disable block comment insertion just for title node, LEAN-2746
-    if ($from.node().type === schema.nodes.title && $from.pos === $to.pos) {
+    if (
```

Comment:
@asouqi 
can you please review this new change ? 
I made this change to prevent inserting content inside section title .

Reply:
you forget to add: `$from.node().type === schema.nodes.section_title`

Reply:
yes sorry !
done